src/rust/sdk/example/src/main.rs

```diff
@@ -0,0 +1,353 @@
+use std::str::FromStr;
```

Comment:
How about a doc comment at the top that tells how to run it?

This should make it possible to run the standalone script without knowledge of the portion of the docs site where it'll eventually be outlined

Something like: https://github.com/econia-labs/econia/blob/6256710b2ebe306d2861e6d02d72f95373500ab6/src/python/sdk/examples/trade.py#L35

Reply:
@CRBl69 bump